pypi/build.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+mkdir -p /build
+cd /build
+
+echo "Hello from PyPi build.sh"
+
+BUILD_DIR=`pwd`
+N_CORES=`nproc`
+
+cp -r /boolector .
+
+# Setup dependencies
+cd boolector
+/bin/sh ./contrib/setup-btor2tools.sh
+/bin/sh ./contrib/setup-cadical.sh
+/bin/sh ./contrib/setup-lingeling.sh
+
+#********************************************************************
+#* boolector
+#********************************************************************
+cd ${BUILD_DIR}
+
+cd boolector
+
+./configure.sh --shared --prefix /usr
+if test $? -ne 0; then exit 1; fi
```

Comment:
Now you’ve got `set -e` at the top you shouldn’t need this check.

Reply:
Yup, looks like I missed removing a couple. Hopefully all clean now